website/docs/r/network_routed_v2.html.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_network_routed_v2"
+sidebar_current: "docs-vcd-resource-network-routed-v2"
+description: |-
+  Provides a VMware Cloud Director Org VDC routed Network. This can be used to create, modify, and
+  delete internal networks for vApps to connect backed by NSX-T and NSX-V.
+---
+
+# vcd\_network\_routed\_v2
+
+Provides a VMware Cloud Director Org VDC routed Network. This can be used to create, modify, and
+delete internal networks for vApps to connect backed by NSX-T and NSX-V.
```

Comment:
Maybe this is only me, but "internal" to me is ambiguous (like in previous comment), esp. when you see it next to vApps...

Reply:
Indeed I have copied this from first iteration  but it somewhat doesn't read well. Changed to second attempt.